web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1192,7 +1192,7 @@ private int executeStatement(boolean silent, String user, String pw, String sql,
 		}
 		finally {
 			try {
-				if (statement != null && !statement.isClosed()) {
```

Comment:
Does this mean that statement.isClosed() is useless?

Reply:
I see you are closing statement on this line https://github.com/openmrs/openmrs-core/blob/48386f10175bf557801706f76c79e1413067673b/web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java#L1179

We can delete that and close it in finally block, so we do not duplicate code

Reply:
OK.

Reply:
I've pushed a fix :)